tests/prague/eip7692_eof_v1/eip7620_eof_create/test_subcontainer_validation.py

```diff
@@ -468,3 +470,89 @@ def test_container_both_kinds_different_sub(eof_test: EOFTestFiller):
             kind=ContainerKind.INITCODE,
         ),
     )
+
+
+@pytest.mark.parametrize(
+    ["deepest_container", "exception"],
+    [
+        pytest.param(Container(sections=[Section.Code(code=Op.STOP)]), None, id="valid"),
+        pytest.param(
+            Container(sections=[Section.Code(code=Op.PUSH0)]),
+            EOFException.MISSING_STOP_OPCODE,
+            id="code-error",
+        ),
+        pytest.param(
+            Container(raw_bytes="EF0100A94F5374FCE5EDBC8E2A8697C15331677E6EBF0B"),
+            EOFException.INVALID_MAGIC,
+            id="structure-error",
+        ),
+    ],
+)
```

Comment:
I also added depest containers with error, mostly to see what error messages look like.